tools/esdlcmd/esdl2ecl.cpp

```diff
@@ -282,6 +282,7 @@ class Esdl2EclCmd : public EsdlConvertCmd
         unsigned start = msTick();
         EsdlIndexedPropertyTrees trees;
         trees.loadFile(srcPath.str(), srcName.str(), srcExt.str(), NULL, optGenerateAllIncludes);
+        //trees.loadFile(srcPath.str(), srcName.str(), srcExt.str(), NULL, true);
```

Comment:
remove commented out code?  Maybe a comment on the line above if something needs explaining